docs/getting-started-guides/azure.md

```diff
@@ -0,0 +1,103 @@
+```
+
+NOTE: This script calls [cluster/kube-up.sh](http://releases.k8s.io/{{page.githubbranch}}/cluster/kube-up.sh)
+which in turn calls [cluster/azure/util.sh](http://releases.k8s.io/{{page.githubbranch}}/cluster/azure/util.sh)
+using [cluster/azure/config-default.sh](http://releases.k8s.io/{{page.githubbranch}}/cluster/azure/config-default.sh).
+
+You must set `AZURE_SUBSCRIPTION_ID` or you willl receive errors. It may be set in `cluster/azure/config-default.sh` or set as environment variables:
+
+```shell
+export AZURE_SUBSCRIPTION_ID="<your subscriptionID (guid)>"
+```
+
+These values can be overriden by setting them in `cluster/azure/config-default.sh` or as environment variables. They are shown here with their default values:
+
+```shell
+export AZURE_DEPLOY_ID="" # autogenerated if blank
+export AZURE_LOCATION="westus"
+export AZURE_RESOURCE_GROUP="" # generated from AZURE_DEPLOY_ID if unset
+export AZURE_MASTER_SIZE="Standard_A1"
+export AZURE_NODE_SIZE="Standard_A1"
+export AZURE_USERNAME="kube"
+export NUM_NODES=3
+export AZURE_AUTH_METHOD="device"
+```
+
+By default, this will deploy a cluster with 4 `Standard_A1`-sized VMs: one master node, three worker nodes. This process takes about 5 to 10 minutes. Once the cluster is up, connection information to the cluster will be displayed. Additionally, your `kubectl` configuration will be updated to know about this cluster and this new cluster will be set as the active context.
+
+The Azure deployment process produces an output directory `cluster/azure/_deployments/${AZURE_DEPLOY_ID}`. In this directory you will find the PKI and SSH assets created for the cluster, as well as a script named `util.sh`. Here are some examples of its usage:
+
+```
+$ cd cluster/azure/_deployments/kube-20160316-001122/
+
+# This uses the client cert with curl to make an http call to the apiserver.
+$ ./util.sh curl api/v1/nodes
+
+# This uses the client cert with kubectl to target this deployment's apiserver.
+$ ./util.sh kubectl get nodes
+
+# This uses the ssh private key to copy the private key itself to the master node.
+$ ./util.sh copykey
+
+# This uses the ssh private key to open an ssh connection to the master.
+$ ./util.sh ssh
+```
+
+### Cluster scaling
+
+The `azkube` tool used internally during `kube-up` can also be used to scale your cluster.
+Here's an example of scaling a default deployment of 3 nodes to 10 nodes:
+
+```
+export AZURE_DEPLOY_ID="kube-20160316-001122"
+$ docker run -it -v "$HOME/.azkube:/.azkube" -v "/tmp:/tmp" \
+    colemickens/azkube:v0.0.2 /opt/azkube/azkube scale \
+    --deployment-name="${AZURE_DEPLOY_ID}" \
+    --node-size="Standard_A1" \
+    --node-count=10
+```
+
+### Cluster tear-down
+
+You can tear-down a cluster using `kube-down.sh`:
+
+```shell
+export AZURE_DEPLOY_ID="kube-20160316-001122"
+$ ./cluster/kube-down.sh
+```
+
+
+### Notes
+
+1. The user account used for these operations must have Contributor access to the Subscription.
```

Comment:
It says 'Owner' in the section at the top, but 'Contributor' here, do they apply to different sets of operations?